python/Makefile

```diff
@@ -12,9 +18,9 @@ generate-protobuf:
 		../schemas/proto/foxglove/*.proto
 	mv foxglove-schemas-protobuf/foxglove_schemas_protobuf/foxglove/* foxglove-schemas-protobuf/foxglove_schemas_protobuf
 	rmdir foxglove-schemas-protobuf/foxglove_schemas_protobuf/foxglove
-	sed -E -i '' 's/from foxglove import/from . import/g' foxglove-schemas-protobuf/foxglove_schemas_protobuf/*_pb2.py
-	sed -E -i '' 's/import foxglove\.(.+)$$/from . import \1 as foxglove_\1/g' foxglove-schemas-protobuf/foxglove_schemas_protobuf/*_pb2.pyi
-	sed -E -i '' 's/foxglove\./foxglove_/g' foxglove-schemas-protobuf/foxglove_schemas_protobuf/*_pb2.pyi
+	sed -E $(SED_ARGS) 's/from foxglove import/from . import/g' foxglove-schemas-protobuf/foxglove_schemas_protobuf/*_pb2.py
```

Comment:
might be a little more bulletproof to write a python script for this?

Reply:
Seems a bit hard 😕 https://stackoverflow.com/a/4427835/23649

Reply:
I'm working on it but it feels not better. Looks kind of like this so far

```python
def main():
    subprocess.check_call(
        [
            "protoc",
            "--python_out=foxglove-schemas-protobuf/foxglove_schemas_protobuf",
            "--mypy_out=foxglove-schemas-protobuf/foxglove_schemas_protobuf",
            "--proto_path=../schemas/proto",
            *glob.glob("../schemas/proto/foxglove/*.proto"),
        ]
    )

    for f in glob.glob(
        "foxglove-schemas-protobuf/foxglove_schemas_protobuf/foxglove/*"
    ):
        shutil.move(f, "foxglove-schemas-protobuf/foxglove_schemas_protobuf")
    shutil.rmtree("foxglove-schemas-protobuf/foxglove_schemas_protobuf/foxglove")

    for f in glob.glob("foxglove-schemas-protobuf/foxglove_schemas_protobuf/*_pb2.py"):
```